assembl/views/api2/content_source.py

```diff
@@ -22,11 +22,17 @@
 def fetch_posts(request):
     ctx = request.context
     csource = ctx._instance
-    force_restart = request.params.get('force_restart', False)
-    reimport = request.params.get('reimport', False)
-    reprocess = request.params.get('reprocess', False)
-    upper_bound = request.params.get('upper_limit', None)
-    lower_bound = request.params.get('lower_limit', None)
+
+    def request_parameter(key, default_value=None):
+        # when client POSTs data as JSON, in Pyramid data does not arrive in request.POST but in request.json_body
+        return request.json_body.get(key, default_value)
```

Comment:
Yes... trying to remember WHY this was sent as JSON, and I got nothing. Maybe I was trying to standardize POST=>body, GET=>query parameters. But even that does not justify Json.
Beginning to think that just maybe we could use query parameters here, or even a standard post body, and simplify everything.